DiscordBot/DiscordCodeAlong.md

```diff
@@ -0,0 +1,280 @@
+# Discord Chat Bot
+
+In this activity, create a Discord chat bot that can respond to messages in a Discord server. How the chat bot responds and what it says is for you to decide! The chat bot could respond to common phrases, or respond to commands with useful features like rolling dice, posting pictures, or acting as a calculator.
+
+## Setting Up
+
+1. Go to [repl.it](https://repl.it)
+1. Log in
+1. Clone the [HyTechClub-Robot-Start repl](https://repl.it/@brebory/HyTechClub-Robot-Start)

```

Comment:
```suggestion
1. Fork the [HyTechClub-Robot-Start repl](https://repl.it/@brebory/HyTechClub-Robot-Start)
```

The repl.it button reads "Fork" not "Clone", so use the more accurate term for clarity.